homeassistant/components/switcher_kis/_service_registration.py

```diff
@@ -0,0 +1,371 @@
+"""Home Assistant Switcher Component.
+
+For controlling the Switcher Boiler Device (https://www.switcher.co.il/).
+Please follow configuring instructions here:
+    https://www.home-assistant.io/components/switcher_kis/
+
+Author: Tomer Figenblat
+
+This script is meant for internal use only.
+Use it for creating and configuring the component.
+"""
+
+from asyncio import TimeoutError as AsyncioTimeoutError
+from asyncio import wait_for
+from datetime import datetime, timedelta
+from logging import getLogger
+from traceback import format_exc
+from typing import Any, Dict, List, Optional  # noqa F401
+
+import voluptuous as vol
+from aioswitcher.bridge import SwitcherV2Thread
```

Comment:
My bad.
I was under the impression that because the installation of the library was called by the component itself, then it will be safe for the platforms to import outside of the functions, seeing it's already been installed.
I will correct this and the following resembling change requests.